twake/backend/node/src/services/messages/services/messages-operations.ts

```diff
@@ -105,6 +108,20 @@ export class ThreadMessagesOperationsService {
       }`,
     );
     await this.repository.save(message, context);
+
+    if ((operation.reactions || []).length === 1) {
+      await this.reactionNotifications.process({
```

Comment:
No, you should not directly call « process » but instead create a rabbit mq task of type notification:reaction